a5/scenarios/6.py

```diff
@@ -0,0 +1,14 @@
+import lib.util as util
+
+
+INTERFACES_FILE = "/etc/network/interfaces"
+
+routing_entries = util.get_default_routing_information()
+default_entry = next((e for e in routing_entries if util.is_default_gateway(e)), None)
+default_iface = default_entry.iface
+
+command = "sed -i 's/dhcp/manual/g' {iface_file}".format(iface_file=INTERFACES_FILE)
```

Comment:
This does worry me a bit, since if the file got modified before then this could just not work (potentially)

Reply:
That's fair I will modify this to be more fine grain by using python i/o instead